test/jdk/sun/security/util/HostnameMatcher/TestHostnameChecker.java

```diff
@@ -193,7 +193,6 @@ public static void main(String[] args) throws Exception {
         check(checker, "altfoo2.com", cert3, true);
         check(checker, "5.6.7.8", cert3, true);
         check(checker, "foo.bar.com", cert4, true);
-        check(checker, "altfoo.bar.com", cert4, true);
```

Comment:
Can expected result be updated to false instead of removing this case?

Reply:
Oh! I see then this is already covered, ignore this comment.

Reply:
It occurred to me that I should probably combine TestHostnameChecker and the Wildcard tests as they are both tests for domain matching. Let me see if it isn't too much work to do that (still waiting for the CSR to be approved anyway).

Reply:
Fixed in latest commit.